schema/types/Answer.js

```diff
@@ -0,0 +1,40 @@
+const { GraphQLObjectType, GraphQLString, GraphQLBoolean, GraphQLInt } = require("graphql");
+
+module.exports = new GraphQLObjectType({
+  name : "Answer",
+  description : "An answer",
+
+  fields : () => ({
+    id : {
+      type: GraphQLInt
```

Comment:
I had that originally, but removed it because graphql implicitly casts IDs to String, and it was breaking my tests. I guess now that it's (essentially) code complete I could go back and do that.